docs/aurora/data-management/daos/daos-overview.md

```diff
@@ -45,154 +66,303 @@ Total size: 6.0 TB
+```
 
-# print your module list (useful for debugging)
-module list
 
-# print your environment (useful for debugging)
-#env
+## NIC and Core Binding
 
-# turn on output of what is executed
-set -x
+Each Aurora compute node has 8 NICs and each DAOS server node has 2 NICs. 
+Each NIC is capable of driving 20-25 GB/s unidirection for data transfer. 
+Every read and write goes over the NIC and hence NIC binding is the key to achieve good performance. 
 
-#
-# clean previous mounts (just in case)
-#
-clean-dfuse.sh ${DAOS_POOL}:${DAOS_CONT}
+For 12 PPN, the following binding is recommended.
 
-# launch dfuse on all compute nodes
-# will be launched using pdsh
-# arguments:
-#   pool:container
-# may list multiple pool:container arguments
-# will be mounted at:
-#   /tmp/\<pool\>/\<container\>
-launch-dfuse.sh ${DAOS_POOL}:${DAOS_CONT}
+```bash
+CPU_BINDING1=list:4:9:14:19:20:25:56:61:66:71:74:79
+```
+![Sample NIC to Core binding](core-NIC-binding.png "Sample NIC to Core binding")
+
+
+
+## Interception library for posix containers
+
+The interception library (IL) is a next step in improving DAOS performance. This provides kernel-bypass for I/O data, leading to improved performance.
+The IL will intercept basic read and write POSIX calls while all metadata calls still go through dFuse.
+The IL can provide a large performance improvement for bulk I/O as it bypasses the kernel and commuNICates with DAOS directly in userspace.
+It will also take advantage of the multiple NICs on the node based on how many MPI processes are running on the node and which CPU socket they are on.
+
+
+
+![Interception library](interception.png "Interception library")
+
+
+
+```bash
+Interception library for POSIX mode 
+
+mpiexec                                            # no interception
+mpiexec --env LD_PRELOAD=/usr/lib64/libioil.so     # only data is intercepted 
+mpiexec --env LD_PRELOAD=/usr/lib64/libpil4dfs.so  # preferred - both metadata and data is intercepted. This provides close to DFS mode performance.
 
-# change to submission directory
-cd $PBS_O_WORKDIR
 
-# run your job(s)
-# these test cases assume 'testfile' is in the CWD
-cd /tmp/${DAOS_POOL}/${DAOS_CONT}
-
-echo "write"
-
-mpiexec -np $((rpn*nnodes)) \
--ppn $rpn \
--d $threads \
---cpu-bind numa \
---no-vni \ # enables DAOS access
--genvall \
-/soft/daos/examples/src/posix-write
-
-echo "read"
-mpiexec -np $((rpn*nnodes)) \
--ppn $rpn \
--d $threads \
---cpu-bind numa \
---no-vni \ # enables DAOS access
--genvall \
-/soft/daos/examples/src/posix-read
-
-# cleanup dfuse mounts
-clean-dfuse.sh ${DAOS_POOL}:${DAOS_CONT}
-
-exit 0
 ```
 
-## Job Submission
 
-The above job script expects two environment variables which you set to the relevant pool and container.
-The `-ldaos=default` switch will ensure that DAOS is available on the compute node.
+## Sample job script
+
+Currently, ``--no-vni`` is required in the ``mpiexec`` command to use DAOS. 
```

Comment:
I added this statement about --no-vni. Please check whether "--no-vni" is still required.